Formula/man-db.rb

```diff
@@ -0,0 +1,78 @@
+class ManDb < Formula
+  desc "Unix documentation system"
+  homepage "http://man-db.nongnu.org/"
+  url "https://download.savannah.gnu.org/releases/man-db/man-db-2.8.5.tar.xz"
+  sha256 "b64d52747534f1fe873b2876eb7f01319985309d5d7da319d2bc52ba1e73f6c1"
+
+  keg_only :provided_by_macos
+
+  depends_on "pkg-config" => :build
+
+  resource "libpipeline" do
+    url "https://download.savannah.gnu.org/releases/libpipeline/libpipeline-1.5.1.tar.gz"
+    sha256 "d633706b7d845f08b42bc66ddbe845d57e726bf89298e2cee29f09577e2f902f"
+  end
+
+  def install
+    resource("libpipeline").stage do
```

Comment:
If we decide to make `libpipeline` as a resource inside `man-db`, we should either,

- configure `libpipeline` to build a static library, `man-db` can then statically link against `libpipeline` and we do not need to install `libpipeline` into the prefix.
- or, change `libpipeline`'s prefix to `#{libexec}/libpipeline`.

Reply:
I'm in favour of using a static library.
```sh
./configure --enable-static --disable-shared
```
should do the trick.

Reply:
`libpipeline` is now build as a static library in the working directory, instead of prefix.